code/modules/admin/verbs/request_music.dm

```diff
@@ -0,0 +1,30 @@
+/mob/verb/request_music()
+	set category = "OOC"
+	set name = "Request Music"
+
+	if(GLOB.say_disabled) //This is here to try to identify lag problems
+		to_chat(usr, span_danger("Speech is currently admin-disabled."), confidential = TRUE)
+		return
+
+	var/msg = tgui_input_text(usr.client, "Please Input a URL", "Music Request", "")
+	//msg = copytext_char(sanitize(msg), 1, MAX_MESSAGE_LEN)
```

Comment:
there's no reason to have this if it's commented out, delete it or uncomment as needed